static/src/questionnaire-detail.js

```diff
@@ -0,0 +1,50 @@
+import Vue from 'vue/dist/vue.js'
+import Dropzone from 'dropzone';
+import Question from './details/Question'
+import Answer from './details/Answer'
+
+
+var url = "/api/question/";
+
+var question_app = new Vue({
+  el: '#question-detail-app',
+  data: {
+    results: {},
+  },
+  methods: {
+    fetchQuestionData: function fetchQuestionData() {
+      var _this = this;
+
+      axios.get(url).then(function (response) {
+        _this.results = response.data;
+        _this.saveResponseFiles(response.data);
+      });
+    },
+    saveResponseFiles: function saveResponseFiles(question_data) {
+      for (var question_id in question_data) {
+        var response_files = question_data[question_id].response_files;
+
+        if (response_files.length == 0) {
+          response_files = null;
```

Comment:
hmmm c'est utile ca? Avec un array vide on a pas le meme resultat? Ca fait pas tres javascriptique :)

Si vraiment c'est necessaire, alors utiliser `undefined` plutot que `null`, parce que `null` a des proprietes cheloues (genre il est de type objet, wtf) : https://stackoverflow.com/questions/500504/why-is-using-for-in-with-array-iteration-a-bad-idea

(et du coup on peut peut etre enlever ce `if`)